assembl/static2/js/app/reducers/rootReducer.js

```diff
@@ -1,17 +1,21 @@
 import { combineReducers } from 'redux';
 import { i18nReducer } from 'react-redux-i18n';
+import client from '../client';
 import Context from './contextReducer';
 import Debate from './debateReducer';
+import Posts from './postsReducer';
 import Partners from './partnersReducer';
 import Synthesis from './synthesisReducer';
 import Auth from './authenticationReducer';
 import Phase from './phaseReducer';
 import Admin from './adminReducer';
 
 export default combineReducers({
+  apollo: client.reducer(),
```

Comment:
There is not really an impact anymore because the Idea component mapStateToProps are simpler now.